jax/_src/debugging.py

```diff
@@ -264,17 +264,34 @@ def _format_print_callback(fmt: str, *args, **kwargs):
 def debug_print(fmt: str, *args, ordered: bool = False, **kwargs) -> None:
   """Prints values and works in staged out JAX functions.
 
-  Note: This function does *not* work with f-strings because the formatting is
-  done lazily.
+  This function does *not* work with f-strings because formatting is delayed.
+  So instead of ``jax.debug.print(f"hello {bar}")``, write
+  ``jax.debug.print("hello {bar}", bar=bar)``.
+
+  This function is a thin convenience wrapper around :func:`jax.debug.callback`.
+  The implementation is essentially
```

Comment:
I think two colons are needed after `essentially` in order for the text that follows it to be rendered as a code block

Reply:
Thanks!